src/subsquare.ts

```diff
@@ -0,0 +1,42 @@
+import fetch from "node-fetch";
+
+interface ReferendaObject {
+  _id: string;
+  referendumIndex: number;
+  title: string;
+  content: string;
+  track: number;
+  createdAt: string;
+  updatedAt: string;
+  lastActivityAt: string;
+  state: {
+    name: "Executed" | string;
+  };
+  onchainData: {
+    /** Compare this to tx.method.hash.toHex() */
+    proposalHash: string;
+    timeline: {
+      _id: string;
+      referendumIndex: number;
+      name: "Executed" | "Confirmed" | "ConfirmStarted" | "DecisionStarted";
+      indexer: {
+        blockHeight: number;
+        /** Use this when generating the comment */
+        blockHash: string;
+        blockTime: number;
+      };
+    }[];
+  };
+}
+
+export class SubsquareApi {
+  private readonly url: string;
+  constructor() {
+    this.url = "https://collectives.subsquare.io";
```

Comment:
It would be consistent to add this to the constants:

https://github.com/paritytech/rfc-action/blob/4d81e1f325181fee9ab8789bbc5394a4268cc7ef/src/constants.ts#L1

Reply:
Good idea! Done!